gimpopenvino/tools/stable-diffusion-ov-server.py

```diff
@@ -232,78 +260,45 @@ def run(model_name,device_name):
                         guidance_scale = data_output["guidance_scale"]
                         strength = data_output["strength"]
                         seed = data_output["seed"]
                         create_gif = False #data_output["create_gif"]
 
-                        if scheduler == "LMSDiscreteScheduler":
-                             #log.info('LMSDiscreteScheduler...')
-                             scheduler = LMSDiscreteScheduler(
-                                    beta_start=0.00085,
-                                    beta_end=0.012,
-                                    beta_schedule="scaled_linear",
 
-                                )
-                        elif scheduler == "PNDMScheduler":
-                            #log.info('PNDMScheduler...')
-                            scheduler = PNDMScheduler(
-
-                                beta_start=0.00085,
-                                beta_end=0.012,
-                                beta_schedule="scaled_linear",
-                                skip_prk_steps = True,
-
-                            )
-
-                        elif scheduler == "LCMScheduler":
-                            #log.info('LCMScheduler...')
-                            scheduler = LCMScheduler(
-                                beta_start=0.00085,
-                                beta_end=0.012,
-                                beta_schedule="scaled_linear"
-                            )
-
-                        elif scheduler == "UniPCMultistepScheduler":
-                            #log.info('UniPCMultistepScheduler')
-                            scheduler = UniPCMultistepScheduler(
-                                beta_start=0.00085,
-                                beta_end=0.012,
-                                beta_schedule="scaled_linear"
-                                )
-
-                        else:
-                             #log.info('EulerDiscreteScheduler...')
-                             scheduler = EulerDiscreteScheduler(
-                             beta_start=0.00085,
-                             beta_end=0.012,
-                             beta_schedule="scaled_linear"
-                             )
 
 
                         strength = 1.0 if init_image is None else strength
                         log.info('Starting inference...')
                         log.info('Prompt: %s',prompt)
                         log.info('negative_prompt: %s',negative_prompt)
                         log.info('num_inference_steps: %s',num_infer_steps)
+                        log.info('num_images: %s',num_images)
                         log.info('guidance_scale: %s',guidance_scale)
                         log.info('strength: %s',strength)
                         log.info('init_image: %s',init_image)
 
+
+
+                        import time
+
+                        #for i in range(num_images):
```

Comment:
remove commented out line that is not needed.

Reply:
done